Source/Utils/WarpXUtil.cpp

```diff
@@ -164,3 +164,47 @@ namespace WarpXUtilIO{
     }
 }
 
+void Store_parserString(std::string query_string,
+                        std::string& stored_string)
```

Comment:
We are using the parser in some other places (to initialize the laser or the plasma density profile). Just curious, could we use this function there too?

Reply:
Using Store_parserString in parseDensity and parseMomentum now.. to store the expression as a string.